lib/Capabilities.php

```diff
@@ -3,13 +3,22 @@
 namespace OCA\Notes;
 
 use OCP\Capabilities\ICapability;
+use OCP\App\IAppManager;
 
 class Capabilities implements ICapability {
 
+	/** @var IAppManager */
+	private $appManager;
+
+	public function __construct(IAppManager $appManager) {
+		$this->appManager = $appManager;
+	}
+
 	public function getCapabilities() {
 		return [
 			'notes' => [
 				'api_version' => Application::$API_VERSIONS,
+				'version' => $this->appManager->getAppVersion('notes'),
```

Comment:
Do I understand this correctly?
1. Nextcloud read the app version when it was installed (because it's used when managing apps)
2. Now the app reads its version back from Nextcloud

Seems a bit roundabout to me. Shouldn't the app version be available in a way similar to the API versions?
Or am I reading this wrong?